docs/multicluster/upgrade.md

```diff
@@ -37,6 +37,43 @@ for the feature in new version.
 It should have no impact during upgrade to those imported resources like Service, Endpoints
 or AntreaClusterNetworkPolicy.
 
+## Upgrade from a version prior to v1.13
+
+Prior to Antrea v1.13, the `ClusterClaim` CRD is used to define both the Cluster ID and
+ClusterSet ID for a ClusterSet. Since Antrea v1.13, the `ClusterClaim` CRD is removed, and
+the `ClusterSet` CRD solely defines a ClusterSet. The name of a ClusterSet CR must match the
+ClusterSet ID, and a new `clusterID` field specifies the Cluster ID.
+
+To upgrade from a version older than v1.13, once you apply the latest manifests for an
+existing ClusterSet, you need to update the existing `ClusterSet` CR with the right `clusterID`
+in the spec. A sample ClusterSet CR is like following:
+
+```yaml
+apiVersion: multicluster.crd.antrea.io/v1alpha2
+kind: ClusterSet
+metadata:
+  name: test-clusterset # This value must match the ClusterSet ID.
+  namespace: kube-system
+spec:
+  clusterID: test-cluster-north # The new added field since v1.13.
+  leaders:
+    - clusterID: test-cluster-north
+      secret: "member-north-token"
+      server: "https://172.18.0.1:6443"
+  namespace: antrea-multicluster
+```
+
+Note: No action is required if there is already a Cluster ID in a `ClusterClaim` CR. The new
+version of Antrea Multi-cluster controller will update the existing `ClusterSet` CR with
+the valid Cluster ID. You can verify this via `kubectl get clusterclaim id.k8s.io -o json -n kube-system | jq -r '.value'`.
+
+Once you finish the cluster upgrade, you should also delete the `ClusterClaim` CRD
+manually, all `ClusterClaim` CRs will be removed automatically after the CRD is deleted.
```

Comment:
"," -> "." or ", and then"
I really hope I need not to remind you again on this.

all existing `ClusterClaim` CRs